circle.yml

```diff
@@ -29,8 +29,7 @@ dependencies:
 # Run flake8 and the Python test suite via nose.
 test:
   override:
-    - docker run -d app:build sh -c 'flake8 . --exclude=app/src && nosetests --exclude-dir=app/src'
-
+    - docker run app:build test
```

Comment:
yes. The `CMD` instruction is overridden by arguments on the command line. The `ENTRYPOINT` is only overridden if you use the `--entrypoint` with `docker run`